packages/main/src/Badge.ts

```diff
@@ -63,6 +87,65 @@ class Badge extends UI5Element {
 	@property({ defaultValue: "1" })
 	colorScheme!: string;
 
+	/**
```

Comment:
is the idea that if design-type is ValueState then valueState will apply, otherwise it will be ignored?
we can discuss in the meeting and if so - it should be explained in the documentation before releasing